config/7.11.0/common.yaml

```diff
@@ -829,6 +829,26 @@ servers:
                       value: "[[.Jvm.GcContainerOptions]]"
                     #[[end]]
                     ## Jvm config END
+                    ## AMQ Streams BEGIN
+                    #[[if .Kafka.ExtEnabled]]
+                    - name: KIE_SERVER_KAFKA_EXT_ENABLED
+                      value: "[[.Kafka.ExtEnabled]]"
+                    - name: KIE_SERVER_KAFKA_EXT_TOPICS
+                      value: "[[.Kafka.Topics]]"
```

Comment:
if the bulk of the kafka fields are optional, it feels like we should have [[if]] statements for each one of these options in the common.yaml. Otherwise, the operator will insert blank values for each one. What would happen to the app if that happened? if these are all required, then that should be changed in the crd.

Reply:
The envs depends on .Kafka.ExtEnabled is is enabled the env will be passed, we are considering all or nothing, depending on the extEnabled

Reply:
so all are required then?

Reply:
@desmax74 any update on this?

Reply:
@spolti is it ok in this way ?

Reply:
yes.